public/scripts/utils.js

```diff
@@ -33,15 +33,15 @@ module.exports = {
   normalizeSearchTerm: function (value) {
     return value
       .toLowerCase()
-      .replace(/à|á|â|ã|ä/g, 'a')
-      .replace(/ç|č|ć/g, 'c')
-      .replace(/è|é|ê|ë/g, 'e')
-      .replace(/ì|í|î|ï/g, 'i')
-      .replace(/ñ|ň|ń/g, 'n')
-      .replace(/ò|ó|ô|õ|ö/g, 'o')
-      .replace(/š|ś/g, 's')
-      .replace(/ù|ú|û|ü/g, 'u')
-      .replace(/ý|ÿ/g, 'y')
-      .replace(/ž|ź/g, 'z');
+      .normalize('NFD')
+      .replace(/[\u0300-\u036f]/g, '')
+      .replace(/đ/g, 'd')
+      .replace(/ħ/g, 'h')
+      .replace(/ı/g, 'i')
+      .replace(/ĸ/g, 'k')
+      .replace(/ŀ/g, 'l')
+      .replace(/ł/g, 'l')
+      .replace(/ß/g, 'ss')
+      .replace(/ŧ/g, 't');
```

Comment:
Following https://github.com/simple-icons/simple-icons/pull/4833 I would turn these around, I think doing the specific replace before normalizing should be more robust against potential changes in the normalize function (though, of course, it should normalize in the same way we are doing manually :upside_down_face:)

```suggestion
      .replace(/đ/g, 'd')
      .replace(/ħ/g, 'h')
      .replace(/ı/g, 'i')
      .replace(/ĸ/g, 'k')
      .replace(/ŀ/g, 'l')
      .replace(/ł/g, 'l')
      .replace(/ß/g, 'ss')
      .replace(/ŧ/g, 't')
      .normalize('NFD')
      .replace(/[\u0300-\u036f]/g, '');
```